newton/_src/utils/import_usd.py

```
@@ -139,7 +140,7 @@ def parse_usd(
     collect_schema_attrs = len(schema_resolvers) > 0
 
     try:
-        from pxr import Sdf, Usd, UsdGeom, UsdPhysics  # noqa: PLC0415
+        from pxr import Sdf, Usd, UsdGeom, UsdPhysics, UsdShade  # noqa: PLC0415
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)